tempesta_fw/http.h

```diff
@@ -484,6 +485,7 @@ struct tfw_http_req_t {
 	TfwHttpCond		cond;
 	TfwStr			userinfo;
 	TfwStr			host;
+	TfwStr			port;
```

Comment:
I've tried to see, why `req->host` doesn't include port part, but it still unclear for me. We should add port subcomponent into `req->host`. This will improve our code a little bit:
- there is frang check that host in uri and 'Host: ' header are equal. The check fails since `req->host` doesn't include port component.
- the http chains rules can use host + port for routing the requests, not host name only
- code in  `tfw_http_prep_redirect()` would be a little bit simpler.

Reply:
I supposed, that we may need a separate value in future for possible checks, but yes - I agree that in context of single-parsed host/port value for `Host` header - it will be more convenient to have host/port value in the single field for URI too. 
I will change this part.

Reply:
Done.